tests/js/spec/views/eventsV2/utils.spec.jsx

```diff
@@ -240,3 +244,274 @@ describe('getFieldRenderer', function() {
+
+    expect(table.columnOrder[0]).toMatchObject({
+      key: 'a(b)',
+      name: 'antbee',
+      aggregation: 'a',
+      field: 'b',
+    });
+    expect(table.columnSortBy[0]).toMatchObject({
+      key: 'a(b)',
+      order: 'desc',
+    });
+  });
+});
+
+describe('encodeColumnOrderAndColumnSortBy', function() {
+  it('can encode 0 elements', function() {
+    const table = {
+      columnOrder: [],
+      columnSortBy: [],
+    };
+
+    const query = encodeColumnOrderAndColumnSortBy(table);
+
+    expect(Array.isArray(query.alias)).toBeTruthy();
+    expect(Array.isArray(query.field)).toBeTruthy();
+    expect(Array.isArray(query.sort)).toBeTruthy();
+    expect(query.alias).toHaveLength(0);
+    expect(query.field).toHaveLength(0);
+    expect(query.sort).toHaveLength(0);
+  });
+
+  it('can encode an array of elements', function() {
+    const table = {
+      columnOrder: [
+        {
+          key: 'a',
+          name: 'ant',
+          aggregation: '',
+          field: 'a',
+        },
+        {
+          key: 'a(b)',
+          name: 'antbee',
+          aggregation: 'a',
+          field: 'b',
+        },
+      ],
+      columnSortBy: [
+        {
+          key: 'a',
+          order: 'asc',
+        },
+        {
+          key: 'a(b)',
+          order: 'desc',
+        },
+      ],
+    };
+
+    const query = encodeColumnOrderAndColumnSortBy(table);
+    expect(Array.isArray(query.alias)).toBeTruthy();
+    expect(Array.isArray(query.field)).toBeTruthy();
+    expect(Array.isArray(query.sort)).toBeTruthy();
+
+    expect(query.alias).toHaveLength(2);
+    expect(query.alias[0]).toBe('ant');
+    expect(query.alias[1]).toBe('antbee');
+
+    expect(query.field).toHaveLength(2);
+    expect(query.field[0]).toBe('a');
+    expect(query.field[1]).toBe('a(b)');
+
+    expect(query.sort).toHaveLength(2);
+    expect(query.sort[0]).toBe('a');
+    expect(query.sort[1]).toBe('-a(b)');
+  });
+
+  it('will build field using "aggregate(field)" when encoding', function() {
+    const table = {
+      columnOrder: [
+        {
+          key: 'someKey',
+          name: 'antbee',
+          aggregation: 'a',
+          field: 'b',
+        },
+      ],
+      columnSortBy: [],
+    };
+
+    const query = encodeColumnOrderAndColumnSortBy(table);
+    expect(Array.isArray(query.alias)).toBeTruthy();
+    expect(Array.isArray(query.field)).toBeTruthy();
+    expect(Array.isArray(query.sort)).toBeTruthy();
+
+    expect(query.alias).toHaveLength(1);
+    expect(query.alias[0]).toBe('antbee');
+
+    expect(query.field).toHaveLength(1);
+    expect(query.field[0]).toBe('a(b)');
+  });
+});
+
+describe('setColumnStateOnLocation', function() {
+  const location = {
+    boba: {
+      fett: 'no',
+      tea: 'yes',
+    },
+    query: {
+      star: {
+        trek: 'maybe',
+        wars: 'perhaps',
+      },
+    },
+  };
+  let browserHistoryPush;
+
+  beforeAll(() => {
+    browserHistoryPush = browserHistory.push;
```

Comment:
No need to do this, you can call `mockRestore` on the jest mock function to restore the original implementation

Reply:
Doesn't our jest config also automatically mock out this function too?